common/djangoapps/terrain/stubs/tests/test_edxnotes.py

```diff
@@ -142,6 +140,19 @@ def test_search(self):
             previous_page=None
         )
 
+    def test_search_usage_ids(self):
+        """
+        Test search with usage ids.
+        """
+        response = requests.get(self._get_url("api/v1/search"), params={
+            "user": "dummy-user-id",
+            "course_id": "dummy-course-id",
+            "usage_id": "dummy-usage-id"
+        })
+        self.assertTrue(response.ok)
+        response = response.json()
+        self.assertEqual(response[0]["usage_id"], "dummy-usage-id")
```

Comment:
@muzaffaryousaf I did not merge these two tests because it would mixup the tests, the first test is for search with paginated response, the latter is when we don't get paginated response.

I can merge the tests if you want, but I think it is better if we separate out the tests based on their cases.